content/applications/hr/lunch/orders.rst

```diff
@@ -0,0 +1,210 @@
+changes to the :guilabel:`Configure Your Order` pop-up window.
+
+.. example::
+   The vendor, The Pizza Palace, provides a free beverage with any purchase. Their products are
+   configured so that a beverage selection is required when adding one of their products to the
+   cart.
+
+   If a selection is **not** made, an error occurs. The message that appears is `You have to
+   order one and only one Free Beverage with Purchase`.
+
+   .. image:: orders/error.png
+      :align: center
+      :alt: The :guilabel:`Validation Error` pop-up window with the specific error for the free
+            beverage displayed.
+
+Your Order summary
+------------------
+
+At the top of the dashboard in the :guilabel:`Your Order` summary, the account information appears
+as well as all the information relating to all orders placed for the day.
+
+As products are added to an order, they appear at the top center of the summary box. Each product is
+listed beneath the words :guilabel:`Your Order`, with the product name, quantity, and a status tag.
+
+The available tags that can be displayed for each item are:
+
+- :guilabel:`To Order`: the product has been added to the cart, but has not been purchased yet.
+- :guilabel:`Ordered`: the product has been purchased by the user and is waiting to be sent to the
+  vendor.
+- :guilabel:`Sent`: the product has been sent to the vendor.
+- :guilabel:`Received`: the product has been delivered by the vendor to the user's location.
+
+Product quantities can be adjusted by clicking the :guilabel:`➕ (plus sign)` or :guilabel:`➖ (minus
+sign)` to the left of the listed product. The product price adjusts in real-time to display the cost
+for the currently selected quantity of the product.
+
+The right side of the :guilabel:`Your Order` summary displays the various purchasing information.
+The :guilabel:`Total` amount for the entire day's lunch order is displayed. The :guilabel:`Already
+Paid` field indicates how much has been paid that day towards the :guilabel:`Total` amount. The
+:guilabel:`To Pay` field displays how much is required to pay to place the currently configured
+order.
+
+.. image:: orders/your-order.png
+   :align: center
+   :alt: The Your Orders section of the dashboard, with the purchasing information highlighted.
+
+.. tip::
+   Users can place multiple orders throughout the day, and are not restricted to placing only one
+   lunch order each day. Multiple orders can be placed due to forgetting to add items to an order,
+   or if there are multiple meals that are available to be purchased for the office, not just lunch.
+
+   Depending on the various vendors and how the vendors and products are configured, it is possible
+   to order breakfast, lunch, dinner, coffee, or snacks.
+
+Submit an order
+---------------
+
+To place the order, click the :guilabel:`Order Now` button on the right-side of the :guilabel:`Your
+Order` summary. The user is charged the amount that is displayed in the :guilabel:`To Pay` field.
```

Comment:
How is the user charged? It seems like magic, I just click order. No POS-- is there an account? running tab?

Can a payment flow be added in?

Reply:
Ill be making a separate doc about the cash/money part of the app, and Ill link it. for now, I added that it is deducted form their Lunch acct balance.